chef/data_bags/crowbar/migrate/nova_dashboard/026_use_keystone_session_timeout.rb

```diff
@@ -0,0 +1,13 @@
+def upgrade ta, td, a, d
+  keystone_instance = a['keystone_instance']
+  proposal_obj = ProposalObject.find_proposal("keystone", keystone_instance)
+  keystone_timeout = proposal_obj["attributes"]["keystone"]["token_expiration"]/60
```

Comment:
@vuntz Good point. I think we could make sure that that doesn't happen by updating the rpm dependency of nova_dashboard on barclamp-keystone to version that provides the attribute.

Still the migration should handle it as well though